datasets/wikisum.yaml

```diff
@@ -0,0 +1,32 @@
+Name: "WikiSum: Coherent Summarization Dataset for EfficientHuman-Evaluation"
+Description: |
+  This dataset provides how-to articles from [wikihow.com](https://www.wikihow.com) and their summaries,
+  written as a coherent paragraph.
+  The dataset itself is available at [wikisum.zip](https://wikisum.s3.amazonaws.com/WikiSumDataset.zip),
+  and contains the article, the summary, the wikihow url, and an official fold (train, val, or test).
+  In addition, human evaluation results are available at
+  [wikisum-human-eval.zip](https://wikisum.s3.amazonaws.com/HumanEvaluation.zip).
+  It consists of human evaluation of the summary of the Pegasus system, annotators response regarding the difficulty
+  of the task, and words they marked as unknown.
+Documentation: https://wikisum.s3.amazonaws.com/README.txt
```

Comment:
It looks like this file doesn't exist right now. Are you planning to upload something there?

Reply:
Uploaded the file. (now it contains just the basic information on the dataset, later we might add a little more usage instructions if needed. )